submit_to_archives.py

```diff
@@ -149,15 +151,20 @@ def get_response_data_from_archive(archive_name, results):
                                       json=payload,
                                       headers=header)
 
-    response_json = archiver_response.json()
+    if archiver_response.status_code != HTTPStatus.OK:
+        logger.info(ERROR_MESSAGE_WENT_WRONG)
+        logger.info(f'HTTP Status code: {archiver_response.status_code}')
+        logger.info(f'HTTP Response: {str(archiver_response.content)}')
+    else:
+        response_json = archiver_response.json()
 
-    logger.info(f'Response: {response_json}')
+        logger.info(f'Response: {response_json}')
 
-    if response_json.get('message', None) is not None:
-        logger.error("Something went wrong while sending data/metadata to archives.")
-        exit(1)
+        if response_json.get('message', None) is not None:
+            logger.error(ERROR_MESSAGE_WENT_WRONG)
+            exit(1)
```

Comment:
I would avoid using exit.
It makes it more difficult to integrate the script later on.